setup.py

```diff
@@ -17,9 +17,14 @@ def readme():
     with open('README.rst') as f:
         return f.read()
 
-INSTALL_REQUIRES = (
-    ['pandas', 'httplib2', 'google-api-python-client', 'oauth2client']
-)
+INSTALL_REQUIRES = [
+    'pandas',
+    'httplib2>=0.9.1',
+    'google-api-python-client>=1.6.2, <2.0.0dev',
+    'google-auth>=1.0.0, <2.0.0dev',
```

Comment:
you need to verify whether these dep packages are avail on conda-forge as well (and if not, prod to have them built there). furthermore unless there is a really really good reason don't put an upper bound on deps.